servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -812,31 +781,7 @@
             "description": "The report of the node health",
             "required": true,
             "schema": {
-              "type": "object",
-              "description": "The report of the node health",
-              "properties": {
-                "SourceId": {
-                  "type": "string"
-                },
-                "Property": {
-                  "type": "string"
-                },
-                "HealthState": {
-                  "$ref": "#/definitions/HealthState"
-                },
-                "Description": {
-                  "type": "string"
-                },
-                "TimeToLiveInMilliSeconds": {
-                  "type": "string"
-                },
-                "SequenceNumber": {
-                  "type": "string"
-                },
-                "RemoveWhenExpired": {
-                  "type": "boolean"
-                }
-              }
+              "$ref": "#/definitions/NodeHealthReport"
```

Comment:
How is the ClusterHealthReport different schemawise when compared with NodeHealthReport.
Why dont you create a generic model named HealthReport and use it everywhere?